graspit.pro

```diff
@@ -58,5 +61,16 @@ win32 {
 }
 
 #------------------GraspIt! core files---------------------
-
 include(graspit-core.pro)
+
+graspit_test{
+	INCLUDEPATH += test/
```

Comment:
Please move this into graspit-lib-linux.pro. Since it seems tests can not (currently) be built under Windows, please add an error message indicating so in graspit-lib-windows.pro